tests/template_tests/filter_tests/test_length.py

```diff
@@ -43,6 +43,16 @@ def test_length07(self):
         output = self.engine.render_to_string('length07', {'None': None})
         self.assertEqual(output, '0')
 
+    @setup({'length08': '{{ dictionary|length }}'})
```

Comment:
Please choose more meaningful names for the template and test methods. The enumeration pattern is due to legacy reordering, as far as I understand.